### PR TITLE
Features movie list

### PR DESCRIPTION
**Pull request by DorotaZielinska**

### CODE REVIEW

**Review by MirekSobota — APPROVED** [review]
Our website looks much better now ! Good job 👌

File: src/common/TileList/index.js [hunk 1]
```diff
@@ -1,21 +1,30 @@
...
-            <div>
-                <StyledTitle>{title}</StyledTitle>
-                <StyledSubbTitle>{year}</StyledSubbTitle>
-                <Genres />
-            </div>
-            <div>
-                <Rating />
-            </div>
+export const TileList = ({ title, year, genres, vote, votes, poster }) => {
+    const getYear = (string) => {
```
Comment (PawelOwiesek) [inline src/common/TileList/index.js:9]: You handled it very cleverly.

File: src/features/MovieList/index.js [hunk 2]
```diff
@@ -26,6 +27,9 @@ const MovieList = () => {
                         />
                     </ItemMoviesList>
                 ))}
+  const popularMovies = useSelector(selectMoviesList);
```
Comment (PawelOwiesek) [inline src/features/MovieList/index.js:30]: I'm afraid this is not a well-resolved conflict, a possible fix in the next branch.

**Review by PawelOwiesek — APPROVED** [review]
Everything looks great, it was easy to implement genres, great work.

**Pull request merged**